core/spec/models/spree/app_configuration_spec.rb

```diff
@@ -32,5 +32,19 @@ class TestPackageFactory; end
       prefs.package_factory.should eq TestPackageFactory
     end
   end
+
+  it 'uses Spree::OrderUpdater by default' do
+    prefs.order_updater_factory = nil
+    prefs.order_updater_factory.should eq Spree::OrderUpdater
+  end
+
+  context 'when an order_updater factory is specified' do
+    class FakeOrderUpdaterFactory; end
+
+    it 'uses the set order_updater factory' do
+      prefs.order_updater_factory = FakeOrderUpdaterFactory
+      prefs.order_updater_factory.should eq FakeOrderUpdaterFactory
+    end
+  end
```

Comment:
> should

I assume that this means the version of rspec that spree depends on doesn't even know about `expect` yet?! 😭

Reply:
I haven't checked but I assume so. My idea is not to diverge too much from the Spree's code style or we'll end up with a Frankenstein. What do you think? Keep in mind this is a 2013's codebase and we could get nuts trying to make it 2018's.